### PR TITLE
Updated AMP start script

### PR DESCRIPTION
**Pull request by m4rkmckenna**
AMP rpm install folders have been updated

### CODE REVIEW

**Comment by tbouron** [comment]
Also tested and LGTM 👍

**Comment by tbouron** [comment]
AMP 4.3 is released, merging

File: files/install_amp.sh [hunk 1]
```diff
@@ -35,22 +37,46 @@ echo "Install AMP"
...
+echo "==                          ▐░▌       ▐░▌▐░▌▐░▌ ▐░▌▐░▌▐░▌       ▐░▌                            =="
+echo "==                          ▐░█▄▄▄▄▄▄▄█░▌▐░▌ ▐░▐░▌ ▐░▌▐░█▄▄▄▄▄▄▄█░▌                            =="
+echo "==                          ▐░░░░░░░░░░░▌▐░▌  ▐░▌  ▐░▌▐░░░░░░░░░░░▌                            =="
+echo "==                          ▐░█▀▀▀▀▀▀▀█░▌▐░▌   ▀   ▐░▌▐░█▀▀▀▀▀▀▀▀▀                             =="
+echo "==                          ▐░▌       ▐░▌▐░▌       ▐░▌▐░▌                                      =="
+echo "==                          ▐░▌       ▐░▌▐░▌       ▐░▌▐░▌                                      =="
+echo "==                          ▐░▌       ▐░▌▐░▌       ▐░▌▐░▌                                      =="
+echo "==                           ▀         ▀  ▀         ▀  ▀                                       =="
+echo "==                                                                                             =="
+echo "==  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄   =="
```
Comment (bostko) [inline files/install_amp.sh]: Nice one :)

File: files/install_amp.sh [hunk 2]
```diff
@@ -35,22 +37,46 @@ echo "Install AMP"
...
+echo "== ▐░█▀▀▀▀▀▀▀▀▀  ▀▀▀▀█░█▀▀▀▀ ▐░█▀▀▀▀▀▀▀█░▌▐░█▀▀▀▀▀▀▀█░▌ ▀▀▀▀█░█▀▀▀▀ ▐░█▀▀▀▀▀▀▀▀▀ ▐░█▀▀▀▀▀▀▀█░▌ =="
+echo "== ▐░▌               ▐░▌     ▐░▌       ▐░▌▐░▌       ▐░▌     ▐░▌     ▐░▌          ▐░▌       ▐░▌ =="
+echo "== ▐░█▄▄▄▄▄▄▄▄▄      ▐░▌     ▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄█░▌     ▐░▌     ▐░█▄▄▄▄▄▄▄▄▄ ▐░▌       ▐░▌ =="
+echo "== ▐░░░░░░░░░░░▌     ▐░▌     ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌     ▐░▌     ▐░░░░░░░░░░░▌▐░▌       ▐░▌ =="
+echo "==  ▀▀▀▀▀▀▀▀▀█░▌     ▐░▌     ▐░█▀▀▀▀▀▀▀█░▌▐░█▀▀▀▀█░█▀▀      ▐░▌     ▐░█▀▀▀▀▀▀▀▀▀ ▐░▌       ▐░▌ =="
+echo "==           ▐░▌     ▐░▌     ▐░▌       ▐░▌▐░▌     ▐░▌       ▐░▌     ▐░▌          ▐░▌       ▐░▌ =="
+echo "==  ▄▄▄▄▄▄▄▄▄█░▌     ▐░▌     ▐░▌       ▐░▌▐░▌      ▐░▌      ▐░▌     ▐░█▄▄▄▄▄▄▄▄▄ ▐░█▄▄▄▄▄▄▄█░▌ =="
+echo "== ▐░░░░░░░░░░░▌     ▐░▌     ▐░▌       ▐░▌▐░▌       ▐░▌     ▐░▌     ▐░░░░░░░░░░░▌▐░░░░░░░░░░▌  =="
+echo "==  ▀▀▀▀▀▀▀▀▀▀▀       ▀       ▀         ▀  ▀         ▀       ▀       ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀   =="
+echo "================================================================================================="
```
Comment (bostko) [inline files/install_amp.sh]: Nice one :)

**Pull request merged**